com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs

```diff
@@ -228,8 +228,6 @@ public class DebugData
             public float fullscreenDebugMip = 0.0f;
             /// <summary>Index of the light used for contact shadows display.</summary>
             public int fullScreenContactShadowLightIndex = 0;
-            /// <summary>XR single pass test mode.</summary>
-            public bool xrSinglePassTestMode = false;
```

Comment:
shouldnt we first deprecate it and redirerect to the new path?

Reply:
I'll re-add it with the Obsolete keyword. There's no alternative for now, but this was only really used internally and not required anymore.